pkg/providers/vsphere/vsphere_test.go

```diff
@@ -3353,3 +3353,40 @@ func TestVsphereProviderMachineConfigsManagedCluster(t *testing.T) {
 		}
 	}
 }
+
+func TestProviderGenerateDeploymentFileForBottleRocketWithNTPConfig(t *testing.T) {
```

Comment:
Can we add another test for ubuntu or redhat?

Reply:
added